apps/feeds/src/app/controllers/country.controller.ts

```diff
@@ -37,17 +36,14 @@ import {
   Query,
   UseInterceptors,
 } from '@nestjs/common';
-import { ApiOkResponse, ApiQuery } from '@nestjs/swagger';
+import { ApiQuery } from '@nestjs/swagger';
 
 @CastcleController({ path: 'metadata', version: '1.0' })
 export class CountryController {
   constructor(private countryService: CountryService) {}
```

Comment:
```suggestion
  constructor(private metadataService: MetadataService) {}
```

use a new metadataService and remove unused country+language services